USERS.md

```diff
@@ -0,0 +1,42 @@
+Who is using Cilium?
+====================
+
+Sharing experiences and learning from other users is essential. We are
+frequently asked who is using a particular feature of Cilium to get in contact
+with other users to share experiences and best-practices. While the Slack
+community allows users to get in touch, it can be challenging to find users of
+a particular feature quickly.
+
+The following is a directory of users to help identify users of individual
+features. The users themselves directly maintain the list.
+
+Adding yourself as a user
+-------------------------
+
+If you are using Cilium, please consider adding yourself as a user with a quick
+description of your use case by opening a pull request to this file and adding
+a section describing your usage of Cilium. If you are open to others contacting
+you about your use of Cilium on Slack, add your Slack nick as well.
+
+    N: Name of user (company or individual)
+    D: Description
+    U: Usage of features
+    Q: Contacts available for questions
```

Comment:
I presume adding information about the scale of the deployment might deprecate too quickly?